smoketest/scripts/cli/test_vpn_ipsec.py

```diff
@@ -265,7 +265,7 @@ def test_site_to_site(self):
 
         # if dpd is not specified it should not be enabled (see T6599)
         swanctl_unexpected_lines = [
-            f'dpd_timeout'
+            f'dpd_timeout',
             f'dpd_delay'
```

Comment:
```suggestion
            'dpd_timeout',
            'dpd_delay',
```

I think it would be good to fix pylint warning also: `F541 [*] f-string without any placeholders`